third-party/dnd-wiki/classes/class-weapon-master.xml

```diff
@@ -0,0 +1,518 @@
+		</sheet>
+		<setters>
+			<set name="sourceUrl">https://www.dandwiki.com/w/index.php?title=Weapon_Master_(5e_Class)&amp;oldid=1131376#Extra_Attack</set>
+		</setters>
+	</element>
+	<element name="Unbeatable Precision" type="Class Feature" source="D&amp;D Wiki" id="ID_WIKI_CLASS_FEATURE_WEAPON_MASTER_UNBEATABLE_PRECISION">
+		<description>
+			<p>Starting at 6th level, your strikes become so graceful, precise and swift, that you can easily target exposed vital organs of a creature before they have enough time to react. Your weapon attack score a critical hit on a roll of a 19 or 20.</p>
+		</description>
+		<sheet>
+			<description>Your weapon attack score a critical hit on a roll of a 19 or 20.</description>
+		</sheet>
+		<setters>
+			<set name="sourceUrl">https://www.dandwiki.com/w/index.php?title=Weapon_Master_(5e_Class)&amp;oldid=1131376#Unbeatable_Precision</set>
+		</setters>
+	</element>
+	<element name="Flawless Stance" type="Class Feature" source="D&amp;D Wiki" id="ID_WIKI_CLASS_FEATURE_WEAPON_MASTER_FLAWLESS_STANCE">
+		<description>
+			<p>At 9th level, you become extremely hard to fight due to your uncanny stance. When a creature you can see or hear attacks you, you can use your reaction to make attacks from that creature roll with disadvantage when attacking you. If the creature misses an attack against you, in the same reaction you can counter attack, making one melee attack against that creature. You can use this feature a number of times equal to your Dexterity modifier(minimum 1) and regain uses of this feature when you finish a long rest.</p>
+		</description>
+		<sheet action="Reaction" usage="{{flawless stance:uses}}/Long Rest">
+			<description>When a creature you can see or hear attacks you, you can use your reaction to make attacks from that creature roll with disadvantage when attacking you. If the creature misses an attack against you, in the same reaction you can counter attack, making one melee attack against that creature.</description>
+		</sheet>
+		<setters>
+			<set name="sourceUrl">https://www.dandwiki.com/w/index.php?title=Weapon_Master_(5e_Class)&amp;oldid=1131376#Flawless_Stance</set>
+		</setters>
+		<rules>
+			<stat name="flawless stance:uses" value="dexterity:modifier" bonus="base" />
+			<stat name="flawless stance:uses" value="1" bonus="base" />
+		</rules>
+	</element>
+	<element name="Adrenaline" type="Class Feature" source="D&amp;D Wiki" id="ID_WIKI_CLASS_FEATURE_WEAPON_MASTER_ADRENALINE">
+		<description>
+			<p>Beginning at 13th level, when you enter combat your body instantly responds to your battle urges releasing a heavy dose of adrenaline into your system. After rolling your Initiative roll, you gain temporary hit points equal to your weapon master level.</p>
+		</description>
+		<sheet>
+			<description>After rolling your Initiative roll, you gain temporary hit points equal to {{level:weapon master}}.</description>
```

Comment:
You can change this:
`you gain temporary hit points equal to {{level:weapon master}}`
to:
`you gain {{level:weapon master}} temporary hit points`